packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -120,6 +120,23 @@ export class PoolPair {
   async testPoolSwap (metadata: TestPoolSwapMetadata): Promise<string> {
     return await this.client.call('testpoolswap', [metadata], 'bignumber')
   }
+
+  /**
+   * Remove pool liquidity transaction
+   *
+   * @param {RemovePoolLiquiditySource} from pool liquidity sources
+   * @param {string | string[]} from[address] provides at least two types of token with format 'amoun@token'
+   * @param {string} shareAddress defi address for crediting tokens
+   * @param {RemovePoolLiquidityOptions} [options]
+   * @param {RemovePoolLiquidityUTXO[]} [options.utxos] utxos array of specific UTXOs to spend
+   * @param {string} [options.utxos.txid]
+   * @param {number} [options.utxos.vout]
+   * @return {Promise<string>}
+   */
+  async removePoolLiquidity (from: RemovePoolLiquiditySource, shareAddress: string, options: RemovePoolLiquidityOptions = {}): Promise<string> {
```

Comment:
```suggestion
  async removePoolLiquidity (address: string , poolAccount: string, options: RemovePoolLiquidityOptions = {}): Promise<string> {
```

Reply:
hmm.. it stated the `return` was undefined
can you push the latest??

Reply:
@canonbrother I'm unable to push due to the errors in `removePoolLiquidity.test`, I did push the other 2 files though.